### PR TITLE
fix offScreen test failing in Chrome

### PR DESCRIPTION
**Pull request by WilcoFiers**
- Errors got swallowed by the browser testrunner. That’s fixed now
- Chrome was failing one of the tests due to not cleaning up scroll position

### CODE REVIEW

File: build/tasks/test-webdriver.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ module.exports = function (grunt) {
 			if (urls.length > 0) {
 				return runTestUrls(driver, urls, errors);
 			} else {
+				driver.quit();
 				return Promise.resolve(errors);
```
Comment (dylanb) [inline build/tasks/test-webdriver.js]: shouldn't you call resolve inside a then after the quite?

```return driver.quit().then(function () {
  Promise.resolve(errors);
});
```

**Comment by marcysutton** [comment]
I think these test timeouts could use your expertise @dylanb.

Also let's be sure to squash this PR when it gets merged since it has commits that don't adhere to our policy.

**Comment by dylanb** [comment]
ok, I fixed the tests - took two changes

The approach I took to syncing the iframe loading could be generalized to all iframe tests by creating a utility function that we include in all iframes that allows us to tell when the iframe has loaded.

**Comment by WilcoFiers** [comment]
Awesome!! Thanks Dylan. That's why you're the iframe king :P

**Pull request merged**